packages/prettier-plugin-liquid/src/printer/print/liquid.ts

```diff
@@ -536,6 +537,30 @@ export function printLiquidDocParam(
   return parts;
 }
 
+export function printLiquidDocExample(
+  path: AstPath<LiquidDocExampleNode>,
+  options: LiquidParserOptions,
+  _print: LiquidPrinter,
+  _args: LiquidPrinterArgs,
+): Doc {
+  const node = path.getValue();
+  const parts: Doc[] = ['@example'];
+
+  if (node.exampleContent?.value) {
+    const content = node.exampleContent.value.trim();
+    if (content) {
+      parts.push(hardline);
+      const lines = content
+        .split('\n')
+        .map((line) => line.trim())
+        .filter(Boolean);
+      parts.push(join(hardline, lines));
```

Comment:
might be able to clean this up depending on how we end up handling the trimming in earlier stages